jedi_remote.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import print_function
+
+import json
+import os
+
+import jedi.api
+
+from subprocess import Popen, PIPE
+
+
+class JediRemote(object):
+    '''Jedi remote process communication client
+
+    This class provide jedi compatible API.
+    '''
+    python = 'python'
+    remote_command = 'jedi-remote-command.py'
+
+    def __init__(self, python=None):
+        if python is not None:
+            self.python = python
+
+        self._process = None
+
+    def __del__(self):
+        if self._process is not None:
+            # XXX: why does this raise exception?
+            #self._process.terminate()
```

Comment:
`Exception AttributeError: "'NoneType' object has no attribute 'SIGTERM'" in <bound method JediRemote.__del__ of <jedi_remote.JediRemote object at 0x7f2f470ed250>> ignored`